judgels-backends/sandalphon/sandalphon-client/src/main/java/judgels/sandalphon/problem/ProblemClient.java

```diff
@@ -0,0 +1,147 @@
+package judgels.sandalphon.problem;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import javax.inject.Named;
+import judgels.sandalphon.SandalphonUtils;
+import judgels.sandalphon.api.SandalphonClientConfiguration;
+import judgels.sandalphon.api.client.problem.ClientProblemService;
+import judgels.sandalphon.api.problem.ProblemInfo;
+import judgels.sandalphon.api.problem.ProblemStatement;
+import judgels.sandalphon.api.problem.bundle.Item;
+import judgels.sandalphon.api.problem.programming.ProblemSubmissionConfig;
+import judgels.sandalphon.api.problem.programming.ProblemWorksheet;
+import judgels.sandalphon.problem.bundle.ItemProcessorRegistry;
+import judgels.service.api.client.BasicAuthHeader;
+
+public class ProblemClient {
```

Comment:
Because the context here is clear ("problem"), I'm suggesting to remove redundant "problemJid" in method names as follows.

Reply:
OK